tests/e2e/specs/factory/NoSetupRepoFactory.spec.ts

```diff
@@ -65,15 +67,23 @@ suite(
 		const timeToRefresh: number = 1500;
 		const changesToCommit: string = new Date().getTime().toString();
 		const fileToChange: string = 'Date.txt';
-		const pushItemLabel: string = 'Push';
-		const commitChangesButtonLabel: string = `Commit Changes on "${FACTORY_TEST_CONSTANTS.TS_SELENIUM_FACTORY_GIT_REPO_BRANCH}"`;
 		const refreshButtonLabel: string = 'Refresh';
-		const label: string = BASE_TEST_CONSTANTS.TS_SELENIUM_PROJECT_ROOT_FILE_NAME;
+		const pushItemLabel: string = 'Push';
 		let testRepoProjectName: string;
 		const isPrivateRepo: string = FACTORY_TEST_CONSTANTS.TS_SELENIUM_IS_PRIVATE_FACTORY_GIT_REPO ? 'private' : 'public';
 
+		async function setupGitConfig(userName: string, userEmail: string): Promise<void> {
+			await userPreferences.openUserPreferencesPage();
+			await userPreferences.openGitConfigPage();
+			await userPreferences.enterGitConfigUserName(userName);
+			await userPreferences.enterGitConfigUserEmail(userEmail);
+			await userPreferences.clickOnGitConfigSaveButton();
+			// await userPreferences.waitGitConfigSaveButtonIsDisabled();// TODO: restore this line after the bug is fixed
```

Comment:
It would be helpful to see the link to the bug ticket in the comment, to be able to uncomment this line once the bug is fixed.